packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -405,6 +426,29 @@ class Auth {
         this.clearStorage()
         return this.loginGuestUser()
     }
+
+    /**
+     * Decode SLAS JWT and extract information such as customer id, usid, etc.
+     *
+     */
+    decodeSlasJWT(jwt: string) {
```

Comment:
```suggestion
    parseSlasJWT(jwt: string) {
```
Just to be pedantic, `decode` implies to me "convert from obfuscated format to readable format". This does the extra step of "convert from obscure three-letter codes to easily understandable properties", so I feel like it should have a different name. 🤷

Reply:
done!